test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/RequestTests.cs

```diff
@@ -420,6 +422,42 @@ public void CanUpgradeRequestWithConnectionKeepAliveUpgradeHeader()
             }
         }
 
+        [Fact]
+        public async Task RequestAbortedTokenFiredOnClientFIN()
+        {
+            var appStarted = new SemaphoreSlim(0);
+            var requestAborted = new SemaphoreSlim(0);
+            var builder = new WebHostBuilder()
+                .UseKestrel()
+                .UseUrls($"http://127.0.0.1:0")
+                .Configure(app => app.Run(async context =>
+                {
+                    appStarted.Release();
+
+                    var token = context.RequestAborted;
+                    token.Register(() => requestAborted.Release());
+
+                    while (!token.IsCancellationRequested)
+                    {
+                        await context.Response.WriteAsync("a");
```

Comment:
Good point. I had just copied this from the original issue, but remove the token arg to `WriteAsync`. But there's no need to write here.